admin/tpl/admin.php

```diff
@@ -67,6 +67,12 @@
 							<button class="button-secondary so-widget-deactivate" data-status="0"><?php esc_html_e( 'Deactivate', 'so-widgets-bundle' ) ?></button>
 						</div>
 
+						<?php
+						$widget_object = !empty( $widget_objects[$file] ) ? $widget_objects[$file] : false;
+						if( !empty( $widget_object ) && $widget_object->initialize_settings_form() ) {
```

Comment:
I would have a method `has_settings()` or maybe `has_global_settings()` to try make it a bit clearer what this is checking for.